frontend/src/components/panels/map_panel/MapPanelContainer.js

```diff
@@ -1,20 +1,77 @@
-import { GoogleApiWrapper, Map } from 'google-maps-react';
-import React, { Component } from 'react';
- 
-export class MapContainer extends Component {
-    render() {
-        return (
-            <Map
-                google={this.props.google}
-                initialCenter={{
-                    lat: 40.854885,
-                    lng: -88.081807
-                }}
-                zoom={15}></Map>
-        )
+import React from 'react';
+import { withScriptjs, withGoogleMap, GoogleMap, Circle } from "react-google-maps";
+import { connect } from 'react-redux';
+import { MeepService } from '../../../services/meep_service';
+import { selectProject } from '../../../actions/project_details';
+import { withRouter} from 'react-router-dom';
+import { selectProjectLocations } from '../../../selectors/locations';
+import { GoogleMapsAPIKey } from '../../../../private/google_maps';
+
+const meep_service = new MeepService();
+
+const mapState = {
+    marker : {
+        strokeColor: "hsl(125, 100%, 14%)",
+        fillColor:'hsl(125, 100%, 14%)',
+        fillOpacity: "0.7",
+        radius: 500,
+        strokeWeight: 1
+    },
+    center: {
+        "lat": 39.0997,
+        "lng": -94.5786
+    },
+    zoom: 10
+}
+
+const mapStateToProps = (state, ownProps) => {
+    return { 
+        ...ownProps,
+        locations: state.locations[0] ? selectProjectLocations(state.locations[0], state.filters) : []
     }
+};
+
+const MyMapComponent = connect(mapStateToProps)(withScriptjs(withGoogleMap((props) => {
+
+    const dispatchProjectSummary = (location) => {
+        meep_service.getProjectDetailsById(location.project_id).then(data => {
+            props.dispatch(selectProject(data));
+            props.history.push("/details");
```

Comment:
```Warning: Please use `require("history").createBrowserHistory` instead of `require("history/createBrowserHistory")`. Support for the latter will be removed in the next major release.```